### DISCUSSION TITLE
Email account used by the Git Hub Campus Expert

### DISCUSSION THREAD

**Discussion by chamodmanithya-tech** [community/community#30481] [GitHub Education] [answered]
Hi GitHub Team,
I am currently a student studying at the Open University of Sri Lanka. I have submitted an application for the Git Hub Campus Expert program.I have added my university email and my personal email account to my Github account. The e-mail account I have used as my primary e-mail account suffered some technical error this evening. I have contacted Google and they are currently working to resolve the issue.

This is what I have to ask you. I am thinking of switching my primary email account to another email account through settings, even if I can't use that email account again. If I do, will the notification go to my old email account if I qualify for the Git Hub Campus Expert program in the future? If not, will the notification go to the new email account?

I hope for a proper explanation for this.
Thank You !

**Comment by cosmicqbit** [comment, marked as answer]
I think Github already mentions this while applying that the notification will be delivered to your primary email account, in your case old email and this actually makes sense. And why changing your primary email post application won't help is because Github takes note of your primary email and saves the same in their student profile database so changing primary email post application won't update the email in Github Campus student profile database. Hope this answers your question.

**Comment by chamodmanithya-tech** [comment]
Thank You So Much !

**Reply by cosmicqbit** [reply]
Please mark it as answered if it qualifies as one. Thank you :)

**Comment by chamodmanithya-tech** [comment]
Sure, I will do that.
